DEVELOPMENT.md

```diff
@@ -2,6 +2,20 @@ Tips for developers
 -------------------
 It's dangerous to go alone. Take these tips in case you need to fit Indico to your particular needs.
 
+## Running a virtual machine
+Install [vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/), install
+[vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest) plugin with `vagrant plugin install vagrant-vbguest`.
+Simply call `vagrant up` to get your virtual machine running and `vagrant ssh` to jump into the virtual machine.
+
+Next steps:
+```
+indico admin user_create -a
```

Comment:
Wouldn't it be easier to just go through the bootstrap form shown when accessing Indico for the first time when there are no users in the database?